templates/cli/lib/commands/deploy.js.twig

```diff
@@ -593,6 +596,93 @@ const deployCollection = async ({ all, yes } = {}) => {
     }
 }
 
+const deployBucket = async ({ all, yes } = {}) => {
+    let response = {};
+
+    let bucketIds = [];
+    const configBuckets = localConfig.getBuckets();
+
+    if(all) {
+        if (configBuckets.length === 0) {
+            throw new Error("No buckets found in the current directory. Run `appwrite init bucket` to fetch all your buckets.");
+        }
+        bucketIds.push(...configBuckets.map((b) => b.$id));
+    }
+
+    if(bucketIds.length <= 0) {
```

Comment:
is it ever possible to be `< 0` ?

Reply:
Nah. It's just a way I do empty array check. Should I do `.length === 0` instead?

Reply:
I think this makes more sense